analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -597,11 +597,12 @@ open class Npm(
         )
 
         fun mapLinesToOrtIssues(prefix: String, severity: Severity) {
-            val issueLines = lines.takeWhile { it.startsWith(prefix) }.mapNotNull { line ->
-                line.removePrefix(prefix).let {
-                    commonSecondaryPrefixes.fold(it) { remainder, prefix -> remainder.removePrefix(prefix) }
-                }.takeUnless { it.isBlank() }
-            }
+            val issueLines = lines.dropWhile { !it.startsWith(prefix) }
```

Comment:
Would move the filtering into the mapNotNull, e.g.:

```
lines.mapNotNull { line ->
    if (!line.startsWith(prefix)) null
    else {
          commonSecondaryPrefixes.fold(line.removePrefix(prefix)) { remainder, prefix ->    
                remainder.removePrefix(prefix) 
           }.takeUnless { it.isBlank() }
     }
}
```

Reply:
Not as (performance) optimization, I meant it as readability optimization, simplification. 
But I understood the code in a wrong way originally.